package.json

```
@@ -235,18 +235,17 @@
     "@react-navigation/native": "^5.9.4",
     "@react-navigation/stack": "^5.14.5",
     "@reduxjs/toolkit": "^1.9.7",
+    "@reown/walletkit": "^1.2.0",
     "@segment/analytics-react-native": "^2.17.0",
     "@segment/sovran-react-native": "^1.0.4",
     "@sentry/integrations": "6.3.1",
     "@sentry/react-native": "^5.33.0",
     "@solana/addresses": "2.0.0",
     "@tradle/react-native-http": "2.0.1",
     "@walletconnect/client": "^1.8.0",
-    "@walletconnect/core": "^2.17.2",
-    "@walletconnect/jsonrpc-types": "^1.0.4",
-    "@walletconnect/react-native-compat": "2.15.0",
-    "@walletconnect/se-sdk": "1.11.0",
-    "@walletconnect/utils": "^2.17.2",
+    "@walletconnect/core": "^2.18.1",
+    "@walletconnect/react-native-compat": "2.18.0",
```

```suggestion
    "@walletconnect/react-native-compat": "2.18.1",
```
to keep all deps synced for consistency